### PR TITLE
feature

### PR DESCRIPTION
**Pull request by BubbleSci**
##### Checklist

- [ ] `npm run test` passes
- [ ] tests and/or benchmarks are included
- [ ] cases or donate is changed or added
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2639) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/wepy?pullRequest=2639) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/wepy?pullRequest=2639) it.</sub>

File: scripts/config.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function getConfig(name) {
     output: {
       file: opt.dest,
       format: opt.format,
-      banner: opt.bannder,
```
Comment (Gcaufy) [inline scripts/config.js:83]: 这个 commit 的 scope 为什么是 use-intercept?

File: README.md [hunk 2]
```diff
@@ -30,18 +30,18 @@ WePY (发音: /'wepi/)是一款让小程序支持组件化开发的框架，通
...
+      color: @color;
+  }
 </style>
 <template>
   <div class="container">
     <div class="num" @tap="num++">
       {{num}}
     </div>
     <div>{{text}}</div>
-    <input v-model="text"></input>
```
Comment (Gcaufy) [inline README.md:44]: commit msg 和这个改动内容有什么关联？

File: packages/use-promisify/README_CN.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+# @wepy/use-promisify
```
Comment (Gcaufy) [inline packages/use-promisify/README_CN.md:1]: 为什么是 fix ?

**Pull request closed without merging**